tikv/raftstore/coprocessor.go

```diff
@@ -334,14 +349,57 @@ func newTableSplitChecker(firstEncodedTablePrefix, splitKey []byte, policy pdpb.
 	}
 }
 
+func isTableKey(encodedKey []byte) bool {
+	if tikv.IsShardingEnabled() {
+		return len(encodedKey) >= tablecodec.TableSplitKeyLen &&
```

Comment:
the `TableSplitKeyLen` is wrong when sharding enabled.
Should be 2 bytes longer than not enabled.
We can define it in this package for now.

Reply:
got it.